pkg/oc/admin/router/router.go

```diff
@@ -466,15 +465,15 @@ func generateProbeConfigForRouter(cfg *RouterConfig, ports []kapi.ContainerPort)
 }
 
 func generateLivenessProbeConfig(cfg *RouterConfig, ports []kapi.ContainerPort) *kapi.Probe {
-	probe := generateProbeConfigForRouter(cfg, ports)
+	probe := generateProbeConfigForRouter("/healthz/ready", cfg, ports)
```

Comment:
There's no way we can use these names without it being utterly confusing.  Your readiness probe is using our canonical liveness probe path, and your liveness probe is using our canonical readiness path.

Change the path you added to the router to `/healthz/controller` (the health of the controller) and that mitigates it.

However, this is still not the correct default configuration.  By default we should be using `/healthz` for liveness.  It's only when you want the "don't bind until loaded" behavior that these should flip.  Unless we switched to "don't bind until loaded" by default in the router and I missed it?